src/App.tsx

```diff
@@ -1,18 +1,37 @@
-import React from 'react';
-
+import React, { useState, useCallback } from 'react';
+import { Tab } from './react-app-env';
 import './App.scss';
+import { Tabs } from './Tabs';
 
 export const tabs: Tab[] = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected tab is&nbsp;Tab 1</h1>
-    <div>{tabs.length}</div>
-  </div>
-);
+const App: React.FC = () => {
+  const [currentTabs] = useState(tabs);
```

Comment:
Since you don't change tabs content, this can be used as simple variable without adding it to State.